wikimedia/wikipedia-ios
# Talk pages empty states and onboarding modal design adjustments

**Phabricator:** https://phabricator.wikimedia.org/T311638#8308578
https://phabricator.wikimedia.org/T311638#8312945

### Notes
* This PR adds some adjustments to padding and image size according to the design review
* It also adds a scroll indicator on the empty view 
* It doesn't address the scroll issues on non-wikipedia talk pages

### Test Steps
1. On the empty talk page and onboarding modal, and check that the design attends the design review comments
2. Check on smaller screens if possible
